<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourcesView</class>
 <widget class="QDialog" name="SourcesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>246</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="StartStopButton" name="startStopButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/start.png</normaloff>:/icons/start.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stepInstructionButton">
       <property name="text">
        <string>Step instruction</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/step-instruction.png</normaloff>:/icons/step-instruction.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stepSubroutineButton">
       <property name="text">
        <string>Step Subroutine</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/step-subroutine.png</normaloff>:/icons/step-subroutine.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ce::CodeEditor" name="codeView">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StartStopButton</class>
   <extends>QPushButton</extends>
   <header>views/StartStopButton.h</header>
  </customwidget>
  <customwidget>
   <class>ce::CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor/CodeEditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startStopButton</sender>
   <signal>clicked()</signal>
   <receiver>SourcesView</receiver>
   <slot>onStartStopButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartStopButtonClicked()</slot>
 </slots>
</ui>
